examples/Joints/Ingen/pinjoint.py

```diff
@@ -0,0 +1,16 @@
+from sympy import Symbol
+from pydy.bodies import Body
+from pydy.joints import PinJoint
+from pydy.joints_method import JointsMethod
+
+parent = Body('parent')
+child = Body('child')
+
+gravity = Symbol('gravity')
+child.add_force((0, child.get_mass() * gravity, 0), child.get_masscenter())
+
+pin_joint = PinJoint('pin_joint', parent, child, parent_point_pos=(1, 1, 1),
+                     child_point_pos=(1, 1, 1))
```

Comment:
I think it would be best to try to create typical systems. Like this one should create a simple pendulum. You can get some ideas from the tests from KanesMethod and LagrangesMethod.